### ISSUE TITLE
node-canvas dependency bug: TypeError: Image or Canvas expected

### ISSUE THREAD

**Issue by jwueller** [jwueller/canvas-long-shadow#1]
This is a bug in the `node-canvas` dependency:
https://github.com/Automattic/node-canvas/issues/487

We'll have to wait for that to be fixed. As a workaround, we are currently exposing our canvas constructor:

```
new CanvasLongShadow.Canvas(width, height)
```

**Comment by joeflateau** [comment]
@jwueller instead of -- or as an additional alternative to -- exposing the instance of Canvas. Would it be possible to ACCEPT an instance of the Canvas module?

**Comment by jwueller** [comment]
@joeflateau Yes, definitely. Would you be willing to do a pull request for adding this feature?

**Comment by joeflateau** [comment]
@jwueller sure, i'm on vacation at this time so it'll be a bit
